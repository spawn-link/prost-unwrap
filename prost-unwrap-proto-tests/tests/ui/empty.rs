mod test {
    prost_unwrap::include!();
}

fn main() {}
