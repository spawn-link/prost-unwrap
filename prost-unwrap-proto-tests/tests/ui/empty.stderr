error: Unexpected syntax: unexpected end of input, expected an expression
 --> tests/ui/empty.rs:2:5
  |
2 |     prost_unwrap::include!();
  |     ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `prost_unwrap::include` (in Nightly builds, run with -Z macro-backtrace for more info)
